OpenRA.Mods.Cnc/Traits/Buildings/ProductionAirdrop.cs

```diff
@@ -29,11 +30,21 @@ public class ProductionAirdropInfo : ProductionInfo
 		[Desc("Cargo aircraft used for delivery. Must have the `Aircraft` trait.")]
 		public readonly string ActorType = "c17";
 
+		[Desc("The cargo aircraft will spawn at the player baseline (map edge closest to the player spawn)")]
+		public readonly bool BaselineSpawn = false;
+
+		[Desc("Direction the aircraft should face to land.")]
+		public readonly int Facing = 64;
+
 		public override object Create(ActorInitializer init) { return new ProductionAirdrop(init, this); }
 	}
 
 	class ProductionAirdrop : Production
 	{
+		CPos startPos;
+		CPos endPos;
+		int spawnFacing;
```

Comment:
These can be local variables.

Reply:
Done.

Reply:
I see you pushed - but I'm not seeing the change. Is github being weird?

Reply:
No, I'm being weird. I pushed before commiting.

Reply:
There we go.